components/03-sections/02-navigation/04-side-navigation-news/side-navigation-news.hbs

```diff
@@ -0,0 +1,91 @@
+<!-- Side Navigation -->
+<div class="sidebar-news">
+    {{!-- Related Stories Links --}}
+    <div class="sidebar-news-container">  
+        <div class="sidebar-news-title">Related News</div>
+            <ul class="related-news-items">
+                <li> <span class="related-news-link-icon"></span> <a href="#" aria-label="Related News Link">Link to another related article or promoted news article a really long title for this one.</a></li>
+                <hr>
+                <li> <span class="related-news-link-icon"></span> <a href="#" aria-label="Related News Link">Link to another related article or promoted news article.</a></li>
+                <hr>
+                <li> <span class="related-news-link-icon"></span> <a href="#" aria-label="Related News Link">Link to another related article.</a></li>
+            </ul>
+    </div>
+    {{!-- END Related Stories Links --}}
+    {{!-- Learn More Links --}}
+    <div class="sidebar-news-container">
+        <div class="sidebar-news-title">Learn More</div>
+        <div class="row align-items-center learn-more-item">
+            <div class="col-3 image-thumbnail">
+                <img src="https://utsa-asc.github.io/college-dls/904-side-navigation-news/college-dls/college/images/News_Blog-1.png" />
+            </div>
+            <div class="col-9">
+                <a href="#" aria-label="Learn More">Link to College, Division, or Provost Initiative a really long title for this one.</a>
```

Comment:
We could probably adjust this when we import it to Cascade, but I don't think it needs an `aria-label` since we only use this if two links have the exact text.